src/domain/checklist/__tests__/uploadToS3Checklist.test.ts

```diff
@@ -0,0 +1,49 @@
+import * as O from 'fp-ts/Option';
+import { PreLoadRecord } from '../../PreLoadRepository';
+import { UploadToS3Record } from '../../UploadToS3RecordRepository';
+import { evalCheck } from '../types';
+import { check0, check1 } from '../uploadToS3Checklist';
+
+const preLoadResponse = { preloadIdx: '0', secret: 'a-secret', url: 'a-url', key: 'a-key' };
+const preLoadBody = { preloadIdx: '0', contentType: 'application/pdf', sha256: 'a-sha256' };
+
+const preLoadRecord: PreLoadRecord = {
+  type: 'PreLoadRecord',
+  input: { apiKey: 'an-api-key', body: [preLoadBody] },
+  output: { statusCode: 200, returned: [preLoadResponse] },
+};
```

Comment:
Maybe we can think to create something like `data.ts` to group some data used within various tests

Reply:
Done :)